maubot/cli/util/spdx.py

```diff
@@ -27,7 +27,7 @@ def load() -> None:
     global spdx_list
     if spdx_list is not None:
         return
-    with pkg_resources.resource_stream("maubot.cli", "res/spdx.json.zip") as disk_file:
+    with resources.open_binary("maubot.cli", "res/spdx.json.zip") as disk_file:
```

Comment:
`open_binary` is deprecated since Python 3.11.

```suggestion
    with resources.files("maubot.cli").joinpath("res/spdx.json.zip").open("rb") as disk_file:
```